src/components/DatePicker/DatePicker.Props.ts

```diff
@@ -67,6 +67,11 @@ export interface IDatePickerProps extends React.Props<DatePicker> {
    * Localized strings to use in the DatePicker
    */
   strings: IDatePickerStrings;
+
+  /**
+  * Locales to use for rendering date strings, in descending order of priority
+  */
+  locales?: string[];
```

Comment:
Also, can we default it to `navigator.language`? Add JSDoc comment if so.

http://stackoverflow.com/questions/673905/best-way-to-determine-users-locale-within-browser